cmd/ssh_usercert_gen/main.go

```diff
@@ -64,9 +65,9 @@ type AppConfigFile struct {
 }
 
 const (
-	AuthLevelNone     = -1
-	AuthLevelPassword = 0
-	AuthLevelU2F      = 1
+	AuthLevelNone     = 0
+	AuthLevelPassword = 1
+	AuthLevelU2F      = 2
```

Comment:
Do you want to differentiate other 2-factor tokens such as VIP Access? If they will all be in the same category, we shouldn't call it "U2F".